RUMM-441 Update the APM and RUM features docs (#258)
DataDog/dd-sdk-android: docs/rum_collection.md

    {{% /tab %}}
    {{< /tabs >}}

Depending on your application's architecture, you can choose one of several implementations of `ViewTrackingStrategy`:

  - `ActivityViewTrackingStrategy`: Every activity in your application is considered a distinct view.
  - `FragmentViewTrackingStrategy`: Every fragment in your application is considered a distinct view.
  - `NavigationViewTrackingStrategy`: If you use the Android Jetpack Navigation library, this is the recommended strategy. It automatically tracks the navigation destination as a distinct view.
  - `MixedViewTrackingStrategy`: Every activity or fragment in your application is considered a distinct view. This strategy is a mix between the `ActivityViewTrackingStrategy` and `FragmentViewTrackingStrategy`.

>   **Note**: For the `ActivityViewTrackingStrategy`, `FragmentViewTrackingStrategy` or `MixedViewTrackingStrategy` you will be able to validate which `Fragment` or `Activity` to be tracked as a RUM View event by providing a `ComponentPredicate` implementation in the constructor.

Reviewer: ```suggestion
  **Note**: For `ActivityViewTrackingStrategy`, `FragmentViewTrackingStrategy`, or `MixedViewTrackingStrategy` you can validate which `Fragment` or `Activity` is tracked as a RUM view event by providing a `ComponentPredicate` implementation in the constructor.
```

>   **Note**: For the `ActivityViewTrackingStrategy`, `FragmentViewTrackingStrategy` or `MixedViewTrackingStrategy` that are more general strategies you will be able to validate which `Fragment` or `Activity` to be tracked as a RUM View event by providing a `ComponentPredicate` implementation in the constructor.

Reviewer 2: you don't need the "that are more general strategies" part.
